bundles/org.openhab.binding.salus/src/main/java/org/openhab/binding/salus/internal/rest/GsonMapper.java

```diff
@@ -161,6 +226,61 @@ public List<DeviceProperty<?>> parseDeviceProperties(String json) {
         return Collections.unmodifiableList(deviceProperties);
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Can you get rid of the suppression?

Reply:
similar to https://github.com/openhab/openhab-addons/pull/16807#discussion_r1616363461